lib/kino/control.ex

```diff
@@ -307,7 +311,10 @@ defmodule Kino.Control do
   def stream(sources) when is_list(sources) do
     for source <- sources, do: assert_stream_source!(source)
 
-    tagged_topics = for %{attrs: %{ref: ref}} <- sources, do: {nil, ref}
+    tagged_topics =
+      for(%{attrs: %{ref: ref}} <- sources, do: {nil, ref}) ++
+        for(%Kino.JS.Live{ref: ref} <- sources, do: {nil, ref})
```

Comment:
This will change the order they are streamed. Could that be an issue? Perhaps a single `for` would be better?

Reply:
What do you mean? We just subscribe to each `ref`, the order is determined by the time each event is published.

Reply:
Ok, so looks good to me. I would still do a single pass (especially because the code is already duplicate), but not a blocker. :)

Reply:
I updated to a single pass including validation :)